Tests/Testo/Invoice/Inv/Exception/PlaywrightRenderFailedExceptionTest.php

```
@@ -0,0 +1,57 @@
+    public function getDetailReturnsRawDetailForLoggingOnly(): void
+    {
+        $exception = new PlaywrightRenderFailedException($this->makeTranslator(), 'node: command not found');
+
+        Assert::same('node: command not found', $exception->getDetail());
+    }
+
+    public function getDetailDefaultsToEmptyString(): void
+    {
+        $exception = new PlaywrightRenderFailedException($this->makeTranslator());
+
+        Assert::same('', $exception->getDetail());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Testo\Assert' in method 'getDetailDefaultsToEmptyString'. (undefined)

(StaticAccess)